fri/test.py

```diff
@@ -77,4 +81,5 @@ def getFilesList():
 # debug()
 # destroy()
 getFilesList()
+# openJupyterLa()
 

```

Comment:
openJupyter

Reply:
Also, pay attention to typos, even if the code works. Codes with typos are hard to maintain.